viadot/sources/velux_club.py

```diff
@@ -0,0 +1,186 @@
+import json
+import urllib
+import os
+from datetime import datetime, timedelta
+
+
+from copy import deepcopy
+from typing import Any, Dict, List, Literal
+import requests
+
+import numpy as np
+import pandas as pd
+
+from ..config import get_source_credentials
+from ..exceptions import CredentialError
+from ..utils import handle_api_response
+from .base import Source
+
+# Customed Errors!
+class Source_NOK(Exception):
```

Comment:
1) Exceptions in Python should be in CamelCase https://peps.python.org/pep-0008/#exception-names
2) Pls add docstrings to explain what the exception is for